core/src/test/scala/filodb.core/store/ChunkSetInfoSpec.scala

```diff
@@ -71,4 +71,20 @@ class ChunkSetInfoSpec extends NativeVectorTest {
       ingestionTime = ingestionTime.plus(1, ChronoUnit.DAYS)
     }
   }
+
+  it("should check min and max bounds") {
```

Comment:
Another unit test to prove:

for all supportedUserTime1, supportedUserTime2 where `supportedUserTime1 < supportedUserTime2` this is always true:

`chunkId(supportedUserTime1) < chunkID(supportedUserTime2)`